arm9/source/dumpOperations.cpp

```diff
@@ -892,10 +892,31 @@ void ndsCardDump(void) {
 				for (u32 i = 0; i < 0x8000; i += 0x200) {
 					cardRead (src+i, copyBuf+i, false);
 				}
-				if (fwrite(copyBuf, 1, (currentSize>=0x8000 ? 0x8000 : currentSize), destinationFile) < 1) {
+
+				if (currentSize < 0x8000) {
+					if (romSize == ndsCardHeader.romSize + 0x88) {
+						// Trimming with RSA, check if it's real
+						u8 *ptr = copyBuf + (ndsCardHeader.romSize % 0x8000);
+						bool hasRsa = false;
+						for (int i = 0; i < 0x88; i++) {
+							if (ptr[i] != 0xFF) {
```

Comment:
do we know for sure this is 0xFF?

I don't remember if it's always 0xFF or it could be 0x0 as well. (Though I may not have been working with a clean dump in my case)

Reply:
I'm pretty sure clean dumps are always solid 0xFF after the ROM, no?

Reply:
Sorry about the long delay, thanks! switched to that now